automation/test.sh

```diff
@@ -281,17 +293,22 @@ ginko_params="--noColor --seed=42"
 
 # Prepare PV for Windows testing
 if [[ $TARGET =~ windows.* ]]; then
+  if [[ $TARGET =~ windows_sysprep.* ]]; then
+    $disk_name = "disk-windows-sysprep"
```

Comment:
Shouldn't this be `disk_name` instead of `$disk_name`?

Reply:
yes